workers/common/src/main/kotlin/common/Extensions.kt

```diff
@@ -75,7 +75,7 @@ fun ConfigManager.readConfigFileWithDefault(
  * This function realizes the contract that if a specific config file is requested, not being able to read it leads to
  * an exception, but the default file is allowed to not exist.
  */
-inline fun <reified T> ConfigManager.getConfigFileWithDefault(
+inline fun <reified T> getConfigFileWithDefault(
```

Comment:
My intention was that this function doesn't leak to the top-level but has to be called on the `ConfigManager` like the other similar functions, because it cannot be made private.

Reply:
If you prefer, I could also revert this, but then suppress the inspection hint about the receiver being unused.

Reply:
I've found [a better solution](https://github.com/eclipse-apoapsis/ort-server/pull/531).